packages/stylelint-plugin-carbon-tokens/README.md

```diff
@@ -0,0 +1,225 @@
+# stylelint-plugin-carbon-tokens
+
+> A stylelint plugin to support the use of carbon component tokens.
+
+## Getting started
+
+To install `stylelint-plugin-carbon-tokens` in your project, you will need to
+run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S stylelint-plugin-carbon-tokens
```

Comment:
this flag is redundant now isn't it? or should this be a dev dependency?

Reply:
Replaced with -D